cmd/xl-v1-multipart.go

```diff
@@ -665,14 +666,22 @@ func (xl xlObjects) CompleteMultipartUpload(ctx context.Context, bucket string,
 		partIdx := objectPartIndex(currentXLMeta.Parts, part.PartNumber)
 		// All parts should have same part number.
 		if partIdx == -1 {
-			logger.LogIf(ctx, InvalidPart{})
-			return oi, InvalidPart{}
+			invp := InvalidPart{
+				PartNumber: partIdx,
```

Comment:
should we log, `part.PartNumber` and `part.ETag`?

Reply:
good idea.